core/common/src/main/java/alluxio/util/io/BufferUtils.java

```diff
@@ -51,6 +53,53 @@ public static int byteToInt(byte b) {
    * this direct buffer should be discarded. This is unsafe operation and currently a work-around to
    * avoid huge memory occupation caused by memory map.
    *
+   * @param buffer bytebuffer
+   */
+  public static synchronized void cleanDirectBuffer(ByteBuffer buffer) {
+    Preconditions.checkNotNull(buffer, "buffer is null");
+    Preconditions.checkArgument(buffer.isDirect(), "buffer isn't a DirectByteBuffer");
+    int javaVersion = Integer.parseInt(System.getProperty("java.version").split("\\D+")[0]);
```

Comment:
I would suggest making this a static constant as this is going to be constant for the life of the JVM.

Also, I've seen in the JVM that this can report `1.8` for java 8, and `11` for java 11. I'm not sure if that would make a difference

Reply:
made it a utils method backed by a static constant.

Reply:
also modifieid it to handle 1.8 or 9-11